rasa/core/policies/ted_policy.py

```diff
@@ -1,3 +1,4 @@
+from __future__ import annotations
```

Comment:
💡 this allows use to get rid of our quoted [forward references](https://www.python.org/dev/peps/pep-0484/#forward-references). E.g. consider the following:

```python
class Foo:
   def __init__(self) -> "Foo":  # the quoted thing is a forward reference as we can't resolve `Foo` yet
     pass
```

becomes 

```python
from __future__ import annotations

class Foo:
   def __init__(self) -> Foo:
     pass
```